tests/functional/src/main/java/org/mule/functional/junit4/ExtensionFunctionalTestCase.java

```diff
@@ -80,7 +80,7 @@
    * @param builders the list of {@link ConfigurationBuilder}s that will be used to initialise the {@link #muleContext}
    */
   @Override
-  protected final void addBuilders(List<ConfigurationBuilder> builders) {
+  protected void addBuilders(List<ConfigurationBuilder> builders) {
```

Comment:
this could be put back when the circualr dependency issue is removed, right? if so, add a TODO for this